apps/essimporter/importer.cpp

```diff
@@ -51,6 +51,7 @@ namespace
         {
             for (int x=0; x<128; ++x)
             {
+                assert(image->data(x,y)+2);
```

Comment:
I spent some time trying that out, but I had no luck. That line as it is won't work, setColor wants a vector of floats. The data being read from the image in the .ess file is unsigned chars. I tried giving setColor this data converted to float (and rearranged into BGR order) but didn't get a correct image from it. Maybe I'm just doing something wrong.

The +2 was unnecessary, you are right, so I'll remove that. I'm inclined to just leave it at that, as all I was trying to do with this PR was silence the warning.